tests/sav/base_init_ret.res

```diff
@@ -0,0 +1 @@
+A
```

Comment:
Did you forget the test/source file?

Reply:
Oh. thanks

![](https://i.imgflip.com/21etlz.jpg)